### ISSUE TITLE
TrackPlayer.updateMetadataForTrack causes app to crash when updated using local file on IOS

### ISSUE THREAD

**Issue by AVATAR197** [react-native-kit/react-native-track-player#944]
**Describe the bug**
When updating artwork via updateMetadataForTrack(id, metadata) method it won't work on android and it causes crash of the app on IOS.

**To Reproduce**
Simply try to update artwork via medatadaForTrack using a file from the assets folder.

```
TrackPlayer.updateMetadataForTrack("1111", {
          title: title == null ? "- - -" : title,
          artist: artist == null ? "- - -" : artist,
          artwork: require('../assets/test.png'),
});
```

**Environment (please complete the following information):**

_Currently, I am using puckeys branch_ 
```
"react-native-track-player": "github:puckey/react-native-track-player#ios-playback-metadata-received",
```
 _so I have support for the **playback-metadata-received** on IOS._ But I had this problem also with current `1.2.3 version`

I am running the app in the simulator

**Code**
```
TrackPlayer.updateMetadataForTrack("1111", {
       title: track,
       artist: artist,
       artwork:  img === "" ? require('../assets/main_logo.png')  : img,
});
```

**When I am updating the artwork using URL it works fine on both platforms.**
```
TrackPlayer.updateMetadataForTrack("1111", {
       title: track,
       artist: artist,
       artwork:  img === "" ?"https://images.pexels.com/photos/1561020/pexels-photo-1561020.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260"  : img,
});
```

**Comment by Guichaguri** [comment]
Let me know if fc1f56616bac59455caf0625614ca712e1ada48c fixes it for you

**Comment by curiousdustin** [comment]
Fix released in [1.2.4](https://github.com/react-native-kit/react-native-track-player/releases/tag/v1.2.4)
